code/ai/aigoals.h

```diff
@@ -84,6 +84,11 @@ struct ai_goal;
 
 #define MAX_AI_DOCK_NAMES				25
 
+#define AI_GOAL_ACHIEVABLE			1
+#define AI_GOAL_NOT_ACHIEVABLE		2
+#define AI_GOAL_NOT_KNOWN			3
+#define AI_GOAL_SATISFIED			4
```

Comment:
As long as these are being moved, can they be turned into `enum class` without too many side effects?

Reply:
Done. The whole area needs a cleanup, replacing int3's with unreachables and asserts instead of if-then-int3's, but that's kinda out of scopefor this I think.